CMakeLists.txt

```diff
@@ -932,12 +933,11 @@ set(QGIS_INSTALL_SYS_LIBS TRUE CACHE BOOL "If set to TRUE install all required s
 
 #############################################################
 # Python build dependency
-
-find_package(PythonLibrary REQUIRED)
-set(MIN_PYTHON_VERSION "3.7")
-if(${PYTHON_SHORT_VERSION} VERSION_LESS ${MIN_PYTHON_VERSION})
-    message(FATAL_ERROR "Python version ${PYTHON_SHORT_VERSION} is too old. Minimum Python version is ${MIN_PYTHON_VERSION}.")
-endif()
+set(Python_FIND_FRAMEWORK "LAST")
+find_package(Python 3.7 REQUIRED COMPONENTS Interpreter)
```

Comment:
can we keep this `3.7` as a variable (MIN_PYTHON_VERSION)? better readability, since I was thinking a bit if it is required or minimum version or what...